modules/ROOT/partials/configuration/crossorigin.adoc

```diff
@@ -0,0 +1,78 @@
+[[crossorigin]]
+== `+crossorigin+`
+
+The `+crossorigin+` option controls how {productname} handles cross-origin resource loading. This option accepts a function that determines the appropriate crossorigin attribute value based on the resource being loaded.
+
+*Type:* `+(url: string, resourceType: 'script' | 'stylesheet') => string+`
+
+*Default value:*
+
+* For {cloudname}: A function that returns `+'anonymous'+` for cloud resources and omits the attribute for other resources.
+* For self-hosted: A function that **omits** the crossorigin attribute for all resources.
+
+*Possible values:* A function that returns one of:
+
+* `+'anonymous'+` - Sets crossorigin="anonymous".
+* `+'use-credentials'+` - Sets crossorigin="use-credentials".
+* `+''+` (empty string) - Omits the crossorigin attribute entirely.
+
+=== Example: Basic configuration
+
+[source,js]
+----
+tinymce.init({
+  selector: "textarea",
+  crossorigin: (url, resourceType) => {
+    // Returning 'anonymous' or 'use-credentials' here would explicitly set the attribute
+    return 'anonymous';
+    // return 'use-credentials';
+    // return undefined; // Omit the 'crossorigin' attribute for all resources by returning undefined
+  },
+});
+----
+
+[IMPORTANT]
+====
+When loading {productname} from {cloudname}:
+
+* The initial script tag must include `+crossorigin="anonymous"+` and `+referrerpolicy="origin"+`.
+* Example:
++
+[source,html,subs="attributes+"]
+----
+<script src="{cdnurl}" referrerpolicy="origin" crossorigin="anonymous"></script>
+----
+====
+
+[TIP]
+====
+* The crossorigin function runs for both scripts and stylesheets loaded by {productname}.
+* Using `+'anonymous'+` sends the Origin header without credentials.
+* Using `+'use-credentials'+` sends the Origin header with credentials (not recommended for most cases).
+* An empty string `''` means no crossorigin attribute will be set, as it omits the attribute entirely for the resource.
+* For stylesheet resources, the `+content_css_cors+` option takes precedence over the `+crossorigin+` function. See: xref:add-css-options.adoc#content_css_cors[content_css_cors] for details about cross-origin stylesheet loading.
+====
+
+=== Setting a global crossorigin function
+
+The crossorigin function can also be set globally using `+tinymce.overrideDefaults()+`:
+
+[source,js]
+----
+tinymce.overrideDefaults({
+  ...tinymce.defaultOptions, // Preserve existing defaults
+  crossorigin: (url, resourceType) => {
+    // Returning 'anonymous' or 'use-credentials' here would explicitly set the attribute
+    return 'anonymous';
+    // return 'use-credentials';
+    // return undefined; // Omit the 'crossorigin' attribute for all resources by returning undefined
+  }
+});
+----
+
+[IMPORTANT]
+====
+When using {cloudname}, preserving the existing defaults is required. Always include `+...tinymce.defaultOptions+` when using `+overrideDefaults+`.
+====
```

Comment:
I'm not sure we really need a global method like this. An integrator could define a single method and use it in two tinymce instances (since the function arguments are not editor-specific).

Also, I think we fixed the thing where `defaultOptions` is required in calls like this. I don't know for sure, but if we avoid `overrideDefaults` we don't need to answer that question today :)

Reply:
Confirm you are happy for this to be removed.
```suggestion
```